force-app/main/default/lwc/crmNavTaskQuickText/crmNavTaskQuickText.js

```diff
@@ -0,0 +1,64 @@
+import { LightningElement, api } from 'lwc';
+import hurtigtekster from '@salesforce/resourceUrl/NKS_NavTaskQuickText';
+import { FlowAttributeChangeEvent } from 'lightning/flowSupport';
+import addBusinessDays from '@salesforce/apex/CRM_DueDateUtility.addBusinessDays';
+
+export default class CrmNavTaskQuickText extends LightningElement {
+    @api selectedValue;
+    selectedValueLabel;
+
+    options = [];
+    items = [];
+
+    connectedCallback() {
+        if (this.items.length === 0) {
+            fetch(hurtigtekster)
+                .then((response) => response.json())
+                .then((data) => {
+                    this.items = data;
+                    this.options = this.items.map((item) => {
+                        return { label: item.name, value: item.hurtigtekst };
+                    });
+                })
+                .catch((error) => {
+                    console.error('Error loading quick text from static resource', error);
+                });
+        }
+    }
+
+    handleChange(event) {
+        const selectedOption = this.options.find((opt) => opt.value === event.detail.value);
+        this.displayedValue = selectedOption.label;
+        this.selectedValue = selectedOption.value;
+        if (this.selectedValue?.includes('{TO_VIRKEDAGER_FRA_DAGENS_DATO}')) {
+            this.calculateAndReplaceBusinessDate(2);
+        } else {
+            this.dispatchAttributeChange();
+        }
+    }
+
+    calculateAndReplaceBusinessDate(numberOfBusinessDays) {
+        addBusinessDays({ numberofBusinessdays: numberOfBusinessDays })
+            .then((result) => {
+                let formattedDate = this.formatDateDDMMYYYY(new Date(result));
+                this.selectedValue = this.selectedValue.replace('{TO_VIRKEDAGER_FRA_DAGENS_DATO}', formattedDate);
```

Comment:
Du kan fortsatt beholde search replace her da, det hadde vært mulig å også legge til en index i json fila, men det virker veldig tungvint for lite performance gain.